src/main/clojure/clara/rules/durability.clj

```diff
@@ -401,60 +401,87 @@
                 :activation-group-fn
                 :alphas-fn))))
 
-(defn create-default-get-alphas-fn [rulebase]
-  (@#'com/create-get-alphas-fn type ancestors rulebase))
+(def ^:private create-get-alphas-fn @#'com/create-get-alphas-fn)
+
+(defn- opts->get-alphas-fn [rulebase opts]
```

Comment:
Yeah, you're right.  I think it is unlikely to override one and not the other, but obviously possible.
Either way, the implementation like what you are saying here makes more sense.  I just didn't put enough thought into this part since it wasn't a usage I was concerned with.